src/extensions/extra/ImageUrlCodeHints/main.js

```diff
@@ -0,0 +1,874 @@
+/*
```

Comment:
What exactly in this file should I review?  What did you write vs. what was from Adobe?